### PR TITLE
[IMP] web, [google_, microsoft_]calendar: improve the calendar's sidebar

### PR DESCRIPTION
**Pull request by edi-odoo**
This PR aims to improve the look of the sidebar in the calendar view and app.
Some spacing was off and some buttons had issues which have been fixed in this PR.
The filter's checkboxes have been harmonized and the calendar's navigation improved 
to look like the one in gantt view.

Enterprise PR : https://github.com/odoo/enterprise/pull/43686

task-3410165
part of task-3326263

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by edi-odoo** [comment]
Hey @Brieuc-brd could you take a look at this PR please ? (and the enterprise one)
Thanks !

File: addons/hr_holidays/static/src/views/calendar/filter_panel/calendar_filter_panel.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
         vertical-align: middle;
     }
 
-    & img {
+    & img:not(.o_avatar) {
         width:30px;
```
Comment (Brieuc-brd) [inline addons/hr_holidays/static/src/views/calendar/filter_panel/calendar_filter_panel.scss]: Space after the property ? As you did here : https://github.com/odoo/odoo/pull/127413/commits/0d60c93e7caf5f23bc6fb06f627b45c07a92f2a9#diff-ede4b1253194882f88cc037247ed60893826c0e8b5b0483b15fd1ecbecc8bce3R4

File: addons/hr_holidays/static/src/views/calendar/filter_panel/calendar_filter_panel.scss [hunk 2]
```diff
@@ -42,4 +27,3 @@
         }
     }
 }
-
```
Comment (Brieuc-brd) [inline addons/hr_holidays/static/src/views/calendar/filter_panel/calendar_filter_panel.scss:45]: I think you should keep this space at the end of the file.

**Review by Brieuc-brd — CHANGES_REQUESTED** [review]
Hey @edi-odoo, thanks for your work 👍 

I've left a few comments below

File: addons/hr_holidays/static/src/views/calendar/filter_panel/calendar_filter_panel.scss (hunk 2 above)
Comment (Brieuc-brd) [reply addons/hr_holidays/static/src/views/calendar/filter_panel/calendar_filter_panel.scss:45]: What about this point ?

File: addons/web/static/src/views/calendar/calendar_controller.scss [hunk 3]
```diff
@@ -34,7 +34,7 @@ $o-cw-filter-avatar-size: 20px;
         }
         // sync buttons are only displayed on calendar.event views
         .o_calendar_sync {
-            padding-top: 0.5em;
+            padding-bottom: 0.5em;
```
[the remark this answers is not in the document]
Comment (Brieuc-brd) [reply addons/web/static/src/views/calendar/calendar_controller.scss:38]: Now that you've added the BS class, you can remove this.

File: addons/web/static/src/views/calendar/filter_panel/calendar_filter_panel.xml [hunk 4]
```diff
@@ -5,55 +5,44 @@
...
-                                    class="position-absolute"
-                                    t-attf-id="o_calendar_filter_{{filterId}}"
-                                    t-att-checked="isAllActive(section)"
-                                    t-on-change="(ev) => this.onAllFilterInputChange(section, ev)"
-                                />
+                        <div
+                            class="o_calendar_filter_items_checkall o-checkbox form-check"
+                            data-value="section"
+                        >
+                            <t t-set="filterId" t-value="nextFilterId" />
```
[the remark this answers is not in the document]
Comment (Brieuc-brd) [reply addons/web/static/src/views/calendar/filter_panel/calendar_filter_panel.xml:17]: What about this point ?

**Review by Brieuc-brd — COMMENTED** [review]
@edi-odoo I added some new comments :)
Also, can you rebase your branch ?

**Comment by edi-odoo** [comment]
👋 @Brieuc-brd 
I've separated the [IMP] commits into a separate PR targeting master here https://github.com/odoo/odoo/pull/127532

**Comment by Brieuc-brd** [comment]
@edi-odoo LGTM 👍 
My only concern is with the PR title/description:

Since `hr_holiday` is no longer edited in this PR, you should update your PR title.

> We also took the opportunity to replace custom SCSS with utility classes.

Not anymore in this PR, right ?

**Comment by Brieuc-brd** [comment]
@edi-odoo Missing `web` in your dfb7ae8899c9f50b41ea3dba8912fd993a23dae0 title.
Maybe you should simplify the title and add the module list in the description :

```
[FIX] web, *: fix buttons squashing and spacing

Impacted modules: web, google_calendar, ...

commit content ...
```

**Comment by edi-odoo** [comment]
@pparidans this one is ready for tech review. Thank you !

**Review by pparidans — APPROVED** [review]
LGTM
@robodoo rebase-ff r+

**Pull request closed without merging**